docs/photoniq/vwrs/03-configuration-and-setup/02-configure-waiting-rooms.md

```diff
@@ -1,25 +1,29 @@
 ---
 sidebar_position: 20
-title: Configuring Waiting Rooms
+title: Configure Waiting Room Parameters
 ---
 
-This page explains what can be configured in each waiting room using API endpoint properties. For more information about how to create a waiting room, refer to [Configure a Domain](configure-domain.md).
+This page explains some parameters to configure when using the VWR using API endpoint properties. These include configurations when:
+
+- Entering the waiting room
+- In the waiting room
+- Exiting the waiting room
 
 ## Entering the Waiting Room
 
 - **Rate Limit:** Set the rate (maximum users per second) at which traffic should be diverted to the waiting room. (`rate_limit` property)
-- **Waiting Room Path:** The URL that the waiting room needs to sit in front of. (`domain_url` property)
+- **Waiting Room Path:** The URL that sits in front of the waiting room (`domain_url` property)
 - **Queue Type:** Defines how requests should be removed from the waiting room, such as FIFO (first in, first out). (`queue_type` property)
```

Comment:
Just point out, here you have it as FIFO, but in other parts of the document you have it as "fifo", lower case.  Lets keep it consistent throughout the document.